tests/operator/operator.go

```diff
@@ -1904,7 +1786,7 @@ spec:
 			}
 
 			By("Verifying all migratable vmi workloads are updated via live migration")
-			verifyVMIsUpdated(migratableVMIs, launcherSha)
+			verifyVMIsUpdated(virtClient, migratableVMIs)
```

Comment:
Can we avoid passing this `virtClient` around? We have one virtClient singleton throughout the test. There is no point in complicating this function's signature with this.